formidable/serializers/forms.py

```diff
@@ -108,23 +123,17 @@ def check_conditions_cohesion(self, data):
                         ids=', '.join(set(missing_fields))
                     )
                 )
-
-        # # 2/ check there is no more than one rule on a field per action
-        # for action, fields_ids in targets_action.items():
-        #     counter = Counter(fields_ids)
-        #     duplicates = [
-        #         field for field, count in counter.items()
-        #         if count > 1
-        #     ]
-        #     if duplicates:
-        #         raise ValidationError(
-        #             'Action {action} in condition ({name}) is used many times for the same fields ({ids})'.format(  # noqa
-        #                 name=condition['name'],
-        #                 action=action,
-        #                 ids=', '.join(duplicates)
-        #             )
-        #         )
-
+            # check field types are valid
+            if len(condition_fields_allowed_types):
```

Comment:
rather than testing for length, it's more Pythonic to just test the variable Falsy-Trueness

```python
if condition_fields_allowed_types:
    # do stuff
```